### PR TITLE
Feature/new elixir 1.4.0

### PR DESCRIPTION
**Pull request by seivan**
* Changed source URL to use Github releases
* Depends on https://github.com/HashNuke/heroku-buildpack-elixir/pull/96

### CODE REVIEW

**Comment by ericmj** [comment]
We want to keep using the releases from S3 because branches and all tags are not available as github releases.

**Comment by seivan** [comment]
@ericmj My bad, when I initially wanted to look for a source I went to Github releases on pure instinct.
Changing it back.

File: lib/elixir_funcs.sh [hunk 1]
```diff
@@ -8,7 +8,8 @@ function download_elixir() {
 
     output_section "Fetching Elixir ${elixir_version}"
 
-    local download_url="https://s3.amazonaws.com/s3.hex.pm/builds/elixir/${elixir_version}.zip"
+
+    local download_url="https://s3.amazonaws.com/s3.hex.pm/builds/elixir/{elixir_version}.zip"
```
Comment (hellomika) [inline lib/elixir_funcs.sh]: This should be `${elixir_version}`.

Comment (seivan) [inline lib/elixir_funcs.sh]: @hellomika Yes it should! Thanks

**Comment by sheldonkreger** [comment]
What's the status of this? Is anybody deploying with 1.4?

**Comment by ericmj** [comment]
Yes. 1.4.0 has been working since it was released.

**Pull request closed without merging**